### PR TITLE
Driver deadman+led+sounds

### PR DESCRIPTION
**Pull request by Cameronni**
Final working code from 2024 pre season (includes LED lights, sounds, working shoulder buttons and driver deadman.

### CODE REVIEW

File: src/main/java/frc/robot/commands/ArcadeDrive.java [hunk 1]
```diff
@@ -42,9 +46,9 @@ public void initialize() {
     // Called every time the scheduler runs while the command is scheduled.
     @Override
     public void execute() {
-
-        double m_left = left.getAsDouble();
-        double m_right = right.getAsDouble();
+        double slowDown = 0.0;
```
Comment (MacPGamesAndApps) [inline src/main/java/frc/robot/commands/ArcadeDrive.java:49]: Is there a reason to keep slowDown if it's set to 0?

Comment (Cameronni) [reply src/main/java/frc/robot/commands/ArcadeDrive.java:49]: I don't believe there is a reason, took a quick look through code and don't remember why its there, doesn't look like it does anything

Comment (MacPGamesAndApps) [reply src/main/java/frc/robot/commands/ArcadeDrive.java:49]: Cool. I'd say we remove it. If it's ever needed, we can fish it back from the Git history :)

**Comment by MacPGamesAndApps** [comment]
The changes are looking good. I'd like @SteveJacka's eyes on it as well, as there are some areas of the code he might be more familiar with than me.

**Comment by SteveJacka** [comment]
I had already gone through it with Cam and we did some clean up before we did the final testing last week…  I’m good with it, will do the merge and delete the unnecessary branches shortly.

**Review by MacPGamesAndApps — APPROVED** [review]
All good.

**Review by SteveJacka — APPROVED** [review]
Looks good

**Comment by SteveJacka** [comment]
Merged and Deleted

**Pull request merged**